standard/grammar.md

```diff
@@ -19,23 +19,23 @@ ASTERISK : '*' ;
 SLASH    : '/' ;
 
 // Source: §7.3.1 General
-Input
-    : Input_Section?
+input
```

Comment:
This file is largely auto-generated on GitHub, the changes here (a) reflect changes elsewhere and (b) are incomplete. Once the GitHub tooling runs they will be replaced. In other words don't review this file!